src/content/docs/troubleshooting.mdx

```diff
@@ -265,6 +265,26 @@ installation. This can be fixed by running any of:
 1. `flutter run` in your project's root directory and targeting an iOS
    simulator/device.
 
+## Unsupported class file major version 65
+
+Sometimes when running `shorebird` commands that involve gradle processes you might face
+this error: `Unsupported class file major version 65`, (or 66, the number varies sometimes).
```

Comment:
Which commands involve Gradle? How can I tell?

Reply:
There are a couple:
 - init
 - doctor
 - all android related
 - maybe others that I don't remember?

Would it be worth to list them all here? I worry they will get outdated, but open to suggestions

Reply:
I don't know if listing them all here is the way to go (we may add more, this will get stale). Our audience has no idea whether the commands they're running use gradle or not. None of our system should assume an understanding of shorebird's implementation details.